client/src/components/ViewPostDialog.tsx

```diff
@@ -354,13 +354,13 @@ export default function ViewPostDialog(props: {
               alignItems: 'center',
             }}
           >
-            <img
-              // TODO src={props.postUser.thumbnail}
-              src='https://i.imgur.com/8EYKtwP.png'
-              alt='Thumbnail'
-              height='400px'
-              width='400px'
-            />
+            {!!props.postUser.thumbnail ? (
+              <img
+                src={props.postUser.thumbnail}
+                alt='Thumbnail'
+                style={{ maxHeight: '60vw', maxWidth: '60vw' }}
```

Comment:
60vw seems too big, I think `30vw` for maxHeight is better. Can you also add padding to the bottom of the image, sometimes the image looks too close to the body text. Just  `pb: 1` should be fine